hnclient-v2/src/main/resources/application.properties

```diff
@@ -14,7 +14,7 @@ max-socket-read-tries=100
 # For HNSecure:
 http-protocol=http
 hnsecure-hostname = localhost
-hnsecure-port = 14885
+hnsecure-port = 14890
```

Comment:
Probably no need to commit this, only changed locally for testing.

Reply:
Reverted port in ce24d4153daacf73c6f52ccfbfec86da003b65ca.